src/rviz/default_plugin/marker_utils.cpp

```diff
@@ -338,12 +338,14 @@ void checkScaleText(const visualization_msgs::Marker& marker, std::stringstream&
     ss << "Text height of TEXT_VIEW_FACING is 0.0 (scale.z).";
     increaseWarningLevel(StatusProperty::Warn, level);
   }
+#if 0  // There is too much code out there, which sets all 3 axis scales
```

Comment:
It's somewhat sad to see this disabled. Still if you must, how about checking instead whether all three scales are equal to ignore "the common issue"?
This would still catch users who set scales independently.

I would still propose to enable the full check for noetic. After all this is almost trivial to fix in code bases.

Reply:
+1 for restricting this to actual issues, ie ignoring x == y == z

Reply:
`rviz` becomes extremely verbose with all this checking enabled. Even a `interactive_markers` utility function sets all three scales.

Reply:
I do see your point :sunglasses: , it probably makes sense to ignore the special case `x==y==z` altogether then.

Still, I would think checking for different values and `x != 0 || y != 0` is a reasonable verification that could expose problems.
After all, the provided information for `x` and/or `y` is silently discarded.

Reply:
I solved the verboseness issue by distinguishing all console debug levels (and not only ok, warn, error as the rviz display does). This way, users can disable the verbose console output by adapting their verbosity level for the newly introduced logger `rviz.marker`.
The scaling check is performed again.